genesis/vis/raytracer.py

```diff
@@ -157,19 +157,11 @@ def __init__(self, options, vis_options):
         self.lights = []
         for light in options.lights:
             light_intensity = light.get("intensity", 1.0)
-            self.lights.append(
```

Comment:
Remove this change if unrelated. If it is, then keep all arguments on multiple lines for readability.